download/releaseNotes/releaseNotes7.adoc

```diff
@@ -5,6 +5,20 @@
 :awestruct-release_notes_version: 7
 :awestruct-release_notes_version_qualifier: Final
 
+== New and noteworthy: Engine 7.32.0.Final
+
+=== `groupBy()` support for constraint streams of all cardinalities
+
+We continue improving the constraint streams API.
```

Comment:
Avoid "we". Use active tense if possible, "you" otherwise.
New and noteworthy should be written in a past tense (not the past tense), because it already happened. We only mention what we did, we are careful with promises/commitments toward the future.

"We continue improving the constraint streams API."
=> "The ConstraintStream API improved futher."